EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+[!!Manage definitions](../Operation/01_ManageDefinitions.md)
+[!!Manage header file settings](../Operation/02_ManageHeaderFileSett.md)
+[!!Manage line item file settings](../Operation/04_ManageLineItemsFileStt.md)
+
+
+# EXPORTS
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+**Exporter definitions**
+
+The *Exporter definitions* view contains a list view of all available exporter definitions. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+The following functions are available for the editing toolbar:
+
+- [x]     
+    Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all exporter definitions in the list are selected.
+
+- [PREVIEW]   
+   Click this button to create a preview of an exporter definition. For detailed information, see [Preview definition](../Operation/01_ManageDefinitions.md#preview-definition). 
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)  
+    Click this button to edit the selected exporter definition, see see [Edit exporter definition](#edit-exporter-definition). This button is only displayed if a single checkbox in the list of exporter definitions is selected. Alternatively, you can directly click a row in the list to edit an exporter definition. The *Edit exporter definition* view is displayed.
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)   
+    Click this button to delete the selected exporter definition, see [Delete definition](../Operation/01_ManageDefinitions.md#delete-definition). This button is only displayed if a single checkbox in the list of exporter definitions is selected.
+   
+
+The following functions and fields are available in this view:
+
+- *Definition name*  
+    Name of the exporter definition.
+
+- *Status*  
```

Comment:
Dummerweise steht hier auch "Active" im UI. :( (Es sollte tatsächlich Status heißen). Was machen wir damit? Es kommt in mehreren Modulen vor. Gerne können wir es besprechen.

Reply:
Müssen wri mal änderung anregen